applications/nftaa.md

```diff
@@ -0,0 +1,411 @@
+ownership.
+This innovative approach opens new avenues for business opportunities and applications in DeFi
+and enterprise blockchain solutions, underscoring our commitment to advancing blockchain technology and its
+applications.
+
+### Project Details
+
+#### Business Overview
+
+NFTs are unique digital assets that can represent a wide range of physical and digital items with
+varying values.
+Traditionally, NFTs operate on the asset side of the owner-asset relationship, lacking the capability to own other
+assets.
+Conversely, accounts represent ownership of assets within the blockchain ecosystem.
+By combining these two concepts, we propose a unique entity that can both be uniquely identifiable and capable of owning
+other assets, leading to the concept of NFTAA.
+
+One of the significant strengths of the Substrate framework is its ability to facilitate the creation of custom
+blockchain logic.
+Through forkless runtime upgrades, it is possible to enhance the functionality of individual modules.
+This flexibility is critical as runtime pallets operate close to the system’s core, offering virtually limitless
+modification options.
+Although the transaction weight is a limiting factor in terms of computing resources available for each transaction, it
+does not detract from the Substrate framework’s overall power and adaptability.
+
+Most Substrate pallets are designed based on a single account architecture, aligning with the NFTAA philosophy and
+facilitating smooth integration with existing infrastructure.
+By treating NFTAAs as conventional accounts, we can easily leverage the existing infrastructure for seamless
+communication.
+Additionally, NFTAAs can function as asset owners or participate in multi-signature schemes, further
+enhancing their utility and security.
+
+<img src="https://raw.githubusercontent.com/decenzio/NFTAA-Polkadot/286414435e0fdd51810c1093688daf67df59d010/proposal/prox.png" alt="Proposal Diagram" width="30%" height="30%">
+
+A significant feature of NFTAA is the ability to transfer ownership of such an account/asset to
+another owner with the real change of the private keys serving the NFTAA.
+For example, if Alice has an NFTAA in her wallet through which she stakes 100 DOT, Alice can sell the NFTAA to Bob,
+which causes Alice's wallet to no longer be able to manipulate the NFTAA,
+meaning, that Bob will have full access using his own private keys that only he knows.
+This gives NFTAA the unique ability to assess many complex use cases.
+
+#### Technical Details
+
+The implementation of the NFTAA concept within the Polkadot ecosystem can be approached in two primary ways.
+The first approach involves creating custom blockchain logic in the form of a pallet, written in Rust as a Substrate
+module.
+The second approach involves using smart contracts. For this proposal, we prefer the pallet approach due to its
+integration capabilities and performance benefits.
```

Comment:
We are glad that you agree with our decision of tech details.
According to our information there are some possibilities to do smart contracts, but we think it's only possible on some parachains (https://wiki.polkadot.network/docs/build-smart-contracts). 
Anyway, if there will be NFTAA pallet in the future it will be possible to add chain extensions and bring this functionality to smart contract layer.

Reply:
I don't see anything about chain-extensions mentioned in the doc and I think this is a deprecated feature. Would be great if you can check on this. Also I think if you were to realize this on the smart-contract level then you can only have this logic be applied on the smart contract level. You wouldn't be able to influence the chain level. Please correct me if I'm wrong.

Reply:
It looks like you are correct, anyway what is stated in the proposition remains valid because to integrate this solution to other pallets, our preferred way is through the pallet approach. But definitely, we will need to freshen up our state of the art in our minds based on our chain extension not knowledge :))))